.gitignore

```diff
@@ -27,6 +27,8 @@ build/Release
 # https://docs.npmjs.com/misc/faq#should-i-check-my-node-modules-folder-into-git
 node_modules
 
+# User-specific stuff:
+.idea/
```

Comment:
IDE ignore?

Reply:
Yeah that's just for webstorm. It adds unnecessary workspace files.